docs/pages/Hosting/LocalTesting.adoc

```diff
@@ -0,0 +1,47 @@
+= Testing netlify deployments locally
```

Comment:
Please make sure, that this is linked in the README.adoc

Reply:
While doing this, it would be great if you could move the "mapping"-chapter under the netlify hosting chapter